psalm.xml

```diff
@@ -16,10 +15,48 @@
         </ignoreFiles>
     </projectFiles>
     <stubs>
+        <file name="vendor/jetbrains/phpstorm-stubs/PDO/PDO.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/ibm_db2/ibm_db2.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/mysqli/mysqli.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/oci8/oci8.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/pgsql/pgsql.php" />
         <file name="vendor/jetbrains/phpstorm-stubs/sqlsrv/sqlsrv.php" />
     </stubs>
+    <issueHandlers>
+        <InvalidParamDefault>
+            <errorLevel type="suppress">
+                <file name="lib/Doctrine/DBAL/Driver/Connection.php" />
+            </errorLevel>
+        </InvalidParamDefault>
```

Comment:
@morozov I don't understand this one… do you?

Reply:
I think it's about the implementors:

https://github.com/doctrine/dbal/blob/35beaca3bed81ad4983e58706ae38688de34a853/lib/Doctrine/DBAL/Connection.php#L823-L826

Reply:
```
ERROR: InvalidParamDefault - lib/Doctrine/DBAL/Driver/Connection.php:35:15 - Default value type null for argument 2 of method Doctrine\DBAL\Connection::quote does not match the given type int (see https://psalm.dev/062)
     * @param int   $type
```
But the default parameter is supposed to be an int, not null…

Reply:
This is what the error is above. The default should be changed to `ParameterType::STRING`. I don't see it being a BC break.

Reply:
Alright let's try that!